pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -77,7 +79,25 @@ This procedure describes how to create a key/value pair secret called `my-secret
    dcos security secrets create --value=top-secret developer/my-secret
    ```
 
-# Creating secrets from a file via the DC/OS Enterprise CLI
+# Creating secrets from a any file via the DC/OS Enterprise CLI
```

Comment:
"a any file" - I guess remove "a"?

Reply:
This title implies that it covers the text file case - as "text" is part of "any".

Reply:
It is true, you can upload text file as a binary secret.